dialect/main.py

```diff
@@ -56,6 +56,7 @@ def do_activate(self):
                 text=self.launch_text,
                 langs=self.launch_langs
             )
+        self.setup_actions()
```

Comment:
```suggestion
            self.setup_actions()
```
Move it inside the window conditional.